api/app/routers/tickets.py

```diff
@@ -0,0 +1,71 @@
+from uuid import UUID
+
+from fastapi import APIRouter, Depends, HTTPException, Query
+from sqlalchemy.orm import Session
+
+from app import models, schemas
+from app.auth.account import get_current_user
+from app.command import get_sorted_paginated_tickets_for_pteams
```

Comment:
```suggestion
from app import command
```
モジュールimportに統一するようルールを決めた